boards/uz7ev_evcc/hdmi/uz7ev_evcc_hdmi.tcl

```diff
@@ -370,12 +408,13 @@ proc avnet_add_hdmi {project projects_folder scriptdir} {
 
   set_property -dict [list CONFIG.PSU__USE__IRQ1 {1}] [get_bd_cells zynq_ultra_ps_e_0]
 
-  create_bd_cell -type ip -vlnv xilinx.com:ip:xlconcat:2.1 xlconcat_1
-  set_property -dict [list CONFIG.NUM_PORTS {1}] [get_bd_cells xlconcat_1]
-  connect_bd_net [get_bd_pins zynq_ultra_ps_e_0/pl_ps_irq1] [get_bd_pins xlconcat_1/dout]
+  set_property -dict [list CONFIG.NUM_PORTS {7}] [get_bd_cells xlconcat_0]
+  #~ create_bd_cell -type ip -vlnv xilinx.com:ip:xlconcat:2.1 xlconcat_1
```

Comment:
Good catch! this xlconcat1 didn't seem to be needed
I think you can remove those lines (instead of commenting them), git history will keep track of this change

Reply:
Comments removed.